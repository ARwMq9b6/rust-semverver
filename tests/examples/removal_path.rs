pub mod old {
    pub mod a {
        pub struct Abc;
    }

    pub mod b {
        pub use old::a::*;
    }

    pub mod c {
        pub use old::a::Abc;
    }
}

pub mod new {
    pub mod a {
        pub struct Abc;
    }
}
