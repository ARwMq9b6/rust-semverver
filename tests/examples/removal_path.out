version bump: 1.0.0 -> (Breaking) -> 2.0.0
warning: Removal of at least one path to `b` (Breaking)
 --> /home/twk/gsoc/rust-semverver/tests/examples/removal_path.rs:7:9
  |
7 | /         pub use old::a::*;
8 | |     }
  | |_____^

warning: Removal of at least one path to `c` (Breaking)
  --> /home/twk/gsoc/rust-semverver/tests/examples/removal_path.rs:11:9
   |
11 | /         pub use old::a::Abc;
12 | |     }
   | |_____^

warning: Removal of at least one path to `Abc` (Breaking)
  --> /home/twk/gsoc/rust-semverver/tests/examples/removal_path.rs:14:13
   |
14 |     pub use self::a::Abc;
   |             ^^^^^^^^^^^^

